src/index.ts

```diff
@@ -50,22 +48,18 @@ export {
     TopicOverloadPolicy,
     HazelcastErrors,
     ReadResultSet,
-    IterationType,
     Aggregators,
-    FieldDefinition,
-    ClassDefinition,
-    ClassDefinitionBuilder,
     EvictionPolicy,
     InMemoryFormat,
     ItemEvent,
-    ItemEventType,
     MapEvent,
     EntryEvent,
     LogLevel,
     JsonStringDeserializationPolicy,
     HazelcastJsonValue,
     LoadBalancer,
     AbstractLoadBalancer,
-    RoundRobinLB,
-    RandomLB,
+    FieldType,
+    SSLOptionsFactory,
+    BasicSSLOptionsFactory,
 };
```

Comment:
At line 18 there is such import `import TopicOverloadPolicy = require('./proxy/topic/TopicOverloadPolicy');`

it renders as a variable definition in generated docs. It would be good to change the way we do import here.

Reply:
Re-exports will be changed a lot in #570, so it won't be a problem.